docs/api/panel.md

```diff
@@ -1,4 +1,14 @@
-# Panel
+---
+weight: 109
```

Comment:
Maybe always put weight at the end? I think it will be better if we keep the same ordering on all files